Velaptor/Observables/OpenGLInitReactor.cs

```diff
@@ -0,0 +1,39 @@
+﻿// <copyright file="OpenGLInitReactor.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace Velaptor.Observables
+{
+    // ReSharper disable RedundantNameQualifier
+    using System.Diagnostics.CodeAnalysis;
+    using Velaptor.Observables.Core;
+    using Velaptor.Observables.ObservableData;
+
+    // ReSharper restore RedundantNameQualifier
+
+    /// <summary>
+    /// Creates an observable to send push notifications of OpenGL events.
```

Comment:
Refactor summary code doc

```suggestion
    /// Creates an observable to send push notification that OpenGL is initialized.
```